Reverts Logic to Fetch Data

AlexCatarino opened the pull request:
Looping through every ticker places a big burden on Quiver API, which is why we changed to the new method of only calling the historical endpoint once.

AlexCatarino committed:
Reverts Logic to Fetch Data

Looping through every ticker places a big burden on Quiver API, which is why we changed to the new method of only calling the historical endpoint once.

Martin-Molinero commented on DataProcessing/QuiverWallStreetBetsDataDownloader.cs:
```diff
@@ -93,121 +86,56 @@ public bool Run()
-                    Task.WaitAll(tasks.ToArray());
-                    
-                    foreach (var kvp in _tempData)
+                    if (_canCreateUniverseFiles && universeCsvContents.Any())
```
nit - `universeCsvContents.Any()` is a proxy for `_canCreateUniverseFiles ` really?

AlexCatarino replied:
Yes, they should have the same result in principle so I added the Boolean first and left the second "to be sure".

Martin-Molinero commented on DataProcessing/QuiverWallStreetBetsDataDownloader.cs:
```diff
@@ -293,96 +204,39 @@ private async Task<string> HttpRequester(string url)
+                lines = lines
+                    .OrderBy(x => DateTime.ParseExact(x.Split(',').First(), "yyyyMMdd", CultureInfo.InvariantCulture,
+                        DateTimeStyles.AdjustToUniversal))
+                    .ToList();
```
Previous implementation was merging with any existing data on disk, current isn't? I think previous would also order universe files which seems like a nice to have behavior?

AlexCatarino replied:
I see your point: if Quiver database removes the old data, we would still save have it. I will also order the universe by SID.

AlexCatarino committed:
Addressed Peer-Review

Re-add logic to `SaveContentToFile` to append information to the old files. Order the universe files alphabetically.
